cmd/inspector/main.go

```diff
@@ -35,6 +36,9 @@ var (
 	// IdentityPath is the path to the identity the inspector should use for network communication
 	IdentityPath = flag.String("identity-path", "", "path to the identity certificate for use on the network")
 
+	// CSVPath is the csv path where command output is written
+	CSVPath = flag.String("csv-path", "stdout", "csv path where command output is written")
```

Comment:
Can you add this only to health command by `healthCmd.Flags().StringVar(&CSVPath, "csv-path", "stdout", "csv path where command output is written")` in `init` method?

Reply:
Why do we even need an output file, people can direct the output a csv as needed.